HelpSource/Classes/String.schelp

```diff
@@ -372,16 +374,23 @@ code::
 ::
 
 method::findRegexp
-POSIX regular expression search.
+Perl regular expression search (see link::Classes/String#Regular expressions::). This method searches exhaustively for matches and collects them into an array of pairs, in the format code::[character index, matching string]::.
+
+"Leftmost largest match": As in the regular expression standard, code::*:: and code::+:: are greedy; if it is possible to have more than one overlapping match for a part of the regular expression, the match list will include only the leftmost and largest of them. In code::"foobar".findRegexp("(o*)(bar)");:: below, code::"o*":: may potentially have three matches: code::"o":: at index 1 (second character), code::"o":: at index 2, and code::"oo":: at index 1. code::findRegexp:: will return only the last of these (code::"oo"::), because it begins in the leftmost-possible matching position, and it is the longest possible match at that position.
```

Comment:
there is no 'regular expression standard'. also this explanation is confusing -- the atom `o*` has 3 matches in `foobar` but only 2 if it's part of the pattern `o*bar`. if you're going to go to the trouble of writing out an example, i think you should choose a clearer one to avoid confusing people.

Reply:
> if you're going to go to the trouble of writing out an example, i think you should choose a clearer one to avoid confusing people.

You had suggested that example :wink: ("(or just annotate the example below for `"foobar".findRegexp("(o*)(bar)");`)")

But I reread your comment -- that suggestion was for marked sub-expressions, which got lost in my explanation.

Rewritten.